react/ProductDetails.js

```diff
@@ -113,157 +109,145 @@ class ProductDetails extends Component {
   }
 
   render() {
-    const { slug, displayVertically } = this.props
+    const { product, loading, displayVertically } = this.props
```

Comment:
Seria interessante o product-details imitar a shelf. Na shelf o que é recebido nas props é o objeto `data`. 
https://github.com/vtex-apps/shelf/blob/master/react/Shelf.js#L64

Reply:
Why? I don´t receive a list of products!

Reply:
👍